draft-thomson-httpbis-http2bis.xml

```diff
@@ -3569,6 +3570,37 @@
             the application protocol and cipher suite are independently selected.
           </t>
         </section>
+        <section anchor="tls13">
+          <name>TLS 1.3 Features</name>
+          <t>
+            TLS 1.3 includes a number of features not available in earlier versions.  This section
+            discusses the use of these features.
+          </t>
+          <t>
+            HTTP/2 servers MUST NOT send post-handshake TLS 1.3 CertificateRequest messages.  HTTP/2
+            clients MUST treat a TLS post-handshake CertificateRequest message as a <xref
+            target="ConnectionErrorHandler">connection error</xref> of type <xref
+            target="PROTOCOL_ERROR" format="none">PROTOCOL_ERROR</xref>.
+          </t>
+          <t>
+            The prohibition on post-handshake authentication applies even if the client offered the
+            "post_handshake_auth" TLS extension.  Post-handshake authentication support might be
+            advertised independently of <xref target="TLS-ALPN">ALPN</xref>.  Clients might offer
+            the capability for use in other protocols, but inclusion of the extension cannot imply
+            support within HTTP/2.
+          </t>
+          <t>
+            <xref target="TLS13"/> defines other post-handshake messages, NewSessionTicket and
+            KeyUpdate, which can be used as they have no direct interaction with HTTP/2.  Unless the
+            use of a new type of TLS message depends on an interaction with the application-layer
+            protocol, that TLS message can be sent after the handshake completes.
+          </t>
+	  <t>
+	    TLS early data MAY be used to send requests, provided that the guidance in <xref
+	    target="RFC8470"/> is observed.  Clients send requests in early data assuming initial
```

Comment:
As this document obsoletes RFC 8470, is it worth importing that guidance into this document wholesale?

Reply:
It's RFC 8740 that this obsoletes :)  Confusing, I know.  I have to double-check every time because I keep getting them the wrong way around.

Reply:
That’s a huge mess.